clinica/pipelines/pet_volume/pet_volume_utils.py

```diff
@@ -23,35 +23,105 @@ def init_input_node(pet_nii):
     return pet_nii
 
 
-def create_binary_mask(tissues, threshold=0.3):
-    from os import getcwd
-    from os.path import basename, join
-
-    import nibabel as nib
-    import numpy as np
-
+def _check_non_empty_tissue_list(tissues: list) -> None:
+    """Check that provided list is non-empty."""
     if len(tissues) == 0:
         raise RuntimeError(
             "The length of the list of tissues must be greater than zero."
         )
 
+
+def _load_tissues(tissues: list):
+    """Aggregates the image data contained in the tissue images provided.
+
+    Parameters
+    ----------
+    tissues : List[PurePath]
+        List of tissue images to aggregate.
+
```

Comment:
The input parameter tissue is of type `list` here you've put `List[PurePath]`

Reply:
Thanks! Addressed in [6833496](https://github.com/aramis-lab/clinica/pull/729/commits/6833496c4a5bf53e3b252598d4adf1c474756960)